integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseBuyDashOrderReviewViewModel.kt

```diff
@@ -43,28 +44,24 @@ class CoinbaseBuyDashOrderReviewViewModel @Inject constructor(
     val showLoading: LiveData<Boolean>
         get() = _showLoading
 
-    val commitBuyOrderFailedCallback = SingleLiveEvent<Unit>()
-
-    private val _transactionCompleted: MutableLiveData<TransactionState> = MutableLiveData()
-    val transactionCompleted: LiveData<TransactionState>
-        get() = _transactionCompleted
+    val commitBuyOrderFailedCallback = SingleLiveEvent<String>()
 
     var sendFundToWalletParams: SendTransactionToWalletParams ? = null
     val placeBuyOrderFailedCallback = SingleLiveEvent<String>()
     private val _placeBuyOrder: MutableLiveData<PlaceBuyOrderUIModel> = MutableLiveData()
     val placeBuyOrder: LiveData<PlaceBuyOrderUIModel>
         get() = _placeBuyOrder
 
-    private val _commitBuyOrderSuccessCallback: MutableLiveData<SendTransactionToWalletParams> = MutableLiveData()
-    val commitBuyOrderSuccessCallback: LiveData<SendTransactionToWalletParams>
+    private val _commitBuyOrderSuccessCallback: MutableLiveData<Event<SendTransactionToWalletParams>> = MutableLiveData()
+    val commitBuyOrderSuccessCallback: LiveData<Event<SendTransactionToWalletParams>>
```

Comment:
Isn't the `SingleLiveEvent` serve the same purpose as `LiveData<Event<>>`? Not sure we need 2 different solutions